server/core/services.py

```
@@ -84,4 +86,134 @@ def logout_user(request):
+            # Create a Project instance using the provided data.
+            # We assume `data` contains "name" and "description".
+            try:
+                project = Project.objects.get(id=data.get("id"))
+                project = data.get('name',project.name)
+                project = data.get('description',project.description)
+                project = data.get('active',project.active)
+                project.save()
+                return project
+
+            except ValidationError as e:
+                    raise ValidationError({'project': e.messages})
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)